src/util/PopulationsMap.test.ts

```diff
@@ -1273,6 +1298,219 @@ it("Adding the observations on clicking the numerator exclusion and denominator
   ).toEqual("uuid-2");
 });
 
+it("NonBool: Adding and Removing the observations to numerator and denominator in Ratio", () => {
+  const ipp: DisplayPopulationValue = {
+    name: PopulationType.INITIAL_POPULATION,
+    expected: 0,
+    actual: false,
+    id: "pid-1",
+    criteriaReference: "",
+  };
+  const denom: DisplayPopulationValue = {
+    name: PopulationType.DENOMINATOR,
+    expected: 1,
+    actual: false,
+    id: "pid-2",
+    criteriaReference: "",
+  };
+
+  const denomExclu: DisplayPopulationValue = {
+    name: PopulationType.DENOMINATOR_EXCLUSION,
+    expected: 0,
+    actual: false,
+    id: "pid-3",
+    criteriaReference: "",
+  };
+  const numer: DisplayPopulationValue = {
+    name: PopulationType.NUMERATOR,
+    expected: 0,
+    actual: false,
+    id: "pid-4",
+    criteriaReference: "",
+  };
+  const numerExclu: DisplayPopulationValue = {
+    name: PopulationType.NUMERATOR_EXCLUSION,
+    expected: 0,
+    actual: false,
+    id: "pid-5",
+    criteriaReference: "",
+  };
+  const populationValues: DisplayPopulationValue[] = [];
+  populationValues.push(ipp);
+  populationValues.push(denom);
+  populationValues.push(denomExclu);
+  populationValues.push(numer);
+  populationValues.push(numerExclu);
```

Comment:
```suggestion
  const populationValues: DisplayPopulationValue[] = [ipp, denom, denomExclu, numer, numerExclu];
```
Pretty sure these are equivalent methods to set the initial array. Optional change.